indico/modules/events/sessions/controllers/display.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+
+from indico.modules.events.models.events import Event
+from indico.modules.events.sessions.models.principals import SessionPrincipal
+from indico.modules.events.sessions.models.sessions import Session
+from indico.modules.events.sessions.views import WPDisplayMySessionsConference
+from MaKaC.webinterface.rh.conferenceDisplay import RHConferenceBaseDisplay
+
+
+class RHSessionListDisplayBase(RHConferenceBaseDisplay):
+    CSRF_ENABLED = True
+
+    def _checkParams(self, params):
+        RHConferenceBaseDisplay._checkParams(self, params)
+        self.event = self._conf
+        self.event_new = Event.find(id=self.event.getId(), is_deleted=False).first_or_404()
```

Comment:
We already have a property for this inherited from `RHConferenceSite`, and also you'd use `conf.as_event` for it